[convai] add silence pricing disclaimer (#1091)
elevenlabs/elevenlabs-docs: fern/conversational-ai/pages/customization/llm/optimising-cost.mdx

Managing Large Language Model (LLM) inference costs is essential for developing sustainable AI applications. This guide outlines key strategies to optimize expenditure on the ElevenLabs platform by effectively utilizing its features. For detailed model capabilities and pricing, refer to our main [LLM documentation](/docs/conversational-ai/customization/llm).

<Note>

>  ElevenLabs supports reducing costs is by reducing inference of the models during

Reviewer: ```suggestion
 ElevenLabs supports reducing costs by reducing inference of the models during
```

>   Another way ElevenLabs supports reducing costs is by reducing inference of our models during

Reviewer: ```suggestion
 ElevenLabs supports reducing costs is by reducing inference of the models during
```

  periods of silence. These periods are billed at 5% of the usual per minute rate. See

>   [here](/docs/conversational-ai/overview#pricing-during-silent-periods) for more details.

Reviewer: ```suggestion
  [the Conversational AI overview page](/docs/conversational-ai/overview#pricing-during-silent-periods) for more details.
```
